icons/detaspace/detaspace-original.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path fill="#ef39a8" d="M117.387 64.031c0 29.469-23.887 53.356-53.356 53.356S10.671 93.5 10.671 64.03s23.892-53.36 53.36-53.36c29.469 0 53.356 23.892 53.356 53.36zm0 0"/><path fill="#bd399c" d="M106.719 64.031c0 23.574-19.114 42.688-42.688 42.688-23.578 0-42.687-19.114-42.687-42.688 0-23.578 19.11-42.687 42.687-42.687 23.574 0 42.688 19.11 42.688 42.687zm0 0"/><path fill="#93388e" d="M96.043 64.031c0 17.68-14.332 32.012-32.012 32.012-17.683 0-32.015-14.332-32.015-32.012 0-17.683 14.332-32.015 32.015-32.015 17.68 0 32.012 14.332 32.012 32.015zm0 0"/><path fill="#563379" d="M85.375 64.031c0 11.785-9.559 21.344-21.344 21.344-11.789 0-21.344-9.559-21.344-21.344 0-11.789 9.555-21.344 21.344-21.344 11.785 0 21.344 9.555 21.344 21.344zm0 0"/></svg>
```

Comment:
For some reason the colors here are different than the ones inside the original-wordmark version. For example the outmost circle here is `#f73b95`, while it is `#f73b95` in the `original-wordmark` version. I'm not sure which one is the correct one, but please make sure to use the same as their official logo :)